mempoolspace/client.go

```diff
@@ -0,0 +1,27 @@
+package mempoolspace
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type Client struct {
+	// meempool parameters
+	minimumFee int64
```

Comment:
The client needs to get the mempool space base uri. In our case: https://mempool.space
Later we can add config file to allow users to use their own mempool space uri.